packages/react-slate-interactive/src/components/ScrollView.js

```diff
@@ -0,0 +1,104 @@
+/* @flow */
+
+import React, { type Element } from 'react';
+import { View } from '@react-slate/core';
+import configureIoHandler from '../io';
+
+type Props = {
+  children: Element<*>,
+  height: number,
+  inverted?: boolean,
+  inputStream: tty$ReadStream,
```

Comment:
Shouldn't this (and `outputStream` and `reportingRatio`) be with `?` as an optional prop as soon it has default props?

Reply:
No, they are required, by if the user doesn't provide 'em, flow will check default props and React will use those.

Reply:
Yep, you are right I've just realized.